.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "common"]
 	path = common
-	url = git@github.com:UWARG/common.git
```

Comment:
@Trotyl15 I think that the url should stay with ssh url because i htink this is what most people use